Header header

float32 thrust
float32 roll
float32 pitch
float32 yaw

float32 kp_roll
float32 kd_roll

float32 kp_pitch
float32 kd_pitch

float32 kp_yaw
float32 kd_yaw

float32 p
float32 q
float32 r

float32 z_correction
float32 r_correction
float32 p_correction